test/unit/googleError.ts

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
Could you also add other error_details beside badrequest? (https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto)
Thank you.